app/src/main/res/layout-land/activity_forgot_password.xml

```diff
@@ -60,7 +60,7 @@
             android:id="@+id/reset_button"
             android:layout_width="@dimen/activity_landscape_width"
             android:layout_height="wrap_content"
-            android:layout_marginTop="16dp"
+            android:layout_marginTop="@dimen/margin_button_large"
```

Comment:
Rather than defining `margin_button_large`, only use set of spacings like `margin_large`

Reply:
Sure @iamareebjamal but here, margin_large was already defined with a value of 10dp hence I had to use a name like that.